src/schema/partner_show.js

```diff
@@ -182,6 +182,13 @@ const PartnerShowType = new GraphQLObjectType({
         return gravity(`partner_show/${id}/images`, options).then(Image.resolve)
       },
     },
+    is_online_exclusive: {
```

Comment:
Is this standard naming style for custom graphql properties? I could do without the `is_` and would expect `onlineExclusive`, but :shrug:.

Reply:
i kind of followed the pattern from other flags like https://github.com/artsy/metaphysics/pull/927/files#diff-787a1e14fd2b358d2ab504851b2d878bL185 bellow it.

Reply:
Related question: We'll probably need to support a new search flag so clients can indicate to elasticsearch that they want to include only shows with real physical locations (e.g., artists' exhibition history). This overlaps with that meaning, so we might want to use the same language. What do you think about something like `has_location: [true|false]` instead of "is online exclusive." The former is closer to the way the data is structured, while the latter is closer to how the shows are presented.

Reply:
Yea, prefixing boolean fields with `is` _is_ some sort of GraphQL convention (will have to dig up where I saw that).

We are totally inconsistent and haven't settled on 🐫 vs 🐍 case, I would be 👍adding new fields as :camel: ?

Reply:
i kind of wanted to have `is_online_exclusive` as its own flag to abstract the logic out of clients, so lets say if tomorrow we decided `online_exclusive` means no location and no date, we'd need to only change the logic in here. 
Good point on the filtering side though, i think that would be an argument probably with the same name?